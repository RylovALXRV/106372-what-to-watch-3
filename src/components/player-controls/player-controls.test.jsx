import React from "react";
import renderer from "react-test-renderer";
import {Provider} from "react-redux";
import configureStore from "redux-mock-store";
import PlayerControls from "./player-controls.jsx";

const mockStore = configureStore([]);

it(`Should PlayerControls render correctly`, () => {
  const store = mockStore({
    progress: 5,
  });

  const tree = renderer.create(
      <Provider store={store}>
        <PlayerControls
          progress={5}
          duration={10}
        />
      </Provider>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
