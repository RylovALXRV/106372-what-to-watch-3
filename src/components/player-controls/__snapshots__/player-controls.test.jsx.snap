// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PlayerControls render correctly 1`] = `
<div
  className="player__controls-row"
>
  <div
    className="player__time"
  >
    <progress
      className="player__progress"
      max="100"
      value={0.8333333333333334}
    />
    <div
      className="player__toggler"
      style={
        Object {
          "left": "0.8333333333333334%",
        }
      }
    >
      Toggler
    </div>
  </div>
  <div
    className="player__time-value"
  >
    00:09:55
  </div>
</div>
`;
